src/lib.rs

```diff
@@ -622,6 +630,20 @@ impl Config {
     pub fn set_disable_active_migration(&mut self, v: bool) {
         self.local_transport_params.disable_active_migration = v;
     }
+
+    /// Sets the congestion control algorithm used (string).
+    ///
+    /// The default value is 'reno'.
+    pub fn set_cc_algorithm(&mut self, name: &str) {
+        self.cc_algorithm = name.to_string();
+    }
+
+    /// Sets the initial congestion control window size (bytes).
+    ///
+    /// The default value is `cc:INITIAL_WINDOW` (10 x `max_datagram_size`).
+    pub fn set_cc_initcwnd(&mut self, v: usize) {
```

Comment:
Why is this needed? I don't think we should let the application pick this, it should just give us what the `max_datagram_size` is. Or if we really need to this should be in number of packets, rather than bytes, so it's lees likely for the application to mess this up.

Maybe we should remove this and move it to a separate PR?

Reply:
Currently there is 2 issues.
1. People/company is playing with initcwnd values, including us, for performance optimization. Also Linux/FreeBSD allow to change initcwnd when socket is created.
2. bytes vs packets - even if we set the number of packet size, it will be converted to actual bytes size -- I am more concerned when we don't know max_datagram_size so actual window size is not known, so choose to set the size as bytes.
Also currently there is a bug #298 also need be aligned with this.

However, separating to another PR is a good idea. I will move to a new PR.